packages/browser-pool/src/puppeteer/puppeteer-plugin.ts

```diff
@@ -74,7 +74,12 @@ export class PuppeteerPlugin extends BrowserPlugin<
                 }
             } catch (error) {
                 await close();
-
+                if (process.env.APIFY_IS_AT_HOME) {
```

Comment:
it feels weird that for playwright we handle this outside of apify platform too, but here we dont. FYI puppeteer now has a similar CLI for installing the browsers https://pptr.dev/browsers-api/

not sure if its already mentioned in the error produced by puppetter?

Reply:
Not really. But the bad thing about this command is that it installs executable into the folder where it was executed, and under a long directory hierarchy, like `chromium/mac_arm/-xxxx.../../.../MacOS/Chromium`, and you have to be aware of that executable location, to e.g. pass it into PUPPETEER_EXECUTABLE_PATH

Reply:
It mentions troubleshooting guide though most of the times